recipe/meta.yaml

```diff
@@ -10,10 +10,11 @@ source:
   url: http://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
   sha256: ec3f5c9714ba0fd45cb4e087301eb1336c317e0d20b575a125050470e8089e4d
   patches:
+    - 349a41da1ad88ad87825414752a8ff5fdd6a6c3f.patch
```

Comment:
Please add an http link to the relevant discussion.  Please also add a note on the anticipated version where this patch can be dropped.